src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="search search__big"
```

Comment:
If you do in this way (when you create a loop as a style), we need some improvements
create a common class like "loop"
```suggestion
        class="search loop"
```